lib/plugin/routes/public/users.js

```diff
@@ -312,10 +312,11 @@ exports.register = function (server, options, next) {
     handler: function (request, reply) {
       const { id } = request.params
       const { organizationId } = request.udaru
-      const { limit, page } = request.query
+      const limit = request.query.limit || server.udaruConfig.get('authorization.defaultPageSize')
```

Comment:
Can this be done with `Joi.default()`? => reusable validation and no extra code needed in the handler 👍